RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,29 @@
+# Release procedure
+
+When you release fixed version of SpotBugs, please follow these procedures.
+
+## Release to Maven Central
+
+Add necessary properties to `gradle.properties` and run `gradle clean build smoketest uploadArchives`.
+Make sure that you have proper `eclipsePlugin/local.properties` file to release Eclipse plugin at the same time.
+
+Check [SonaType official page](http://central.sonatype.org/pages/gradle.html) for detail.
+
+## Release to Eclipse Marketplace
+
+TBU
+
+## Release to Eclipse Update Site
+
+Send pull-request to spotbugs/spotbugs.github.io, to update contents.
+As files to upload, you can use the zip file which is uploaded to Maven Central when you release to there.
+
+See [this pull-request](https://github.com/spotbugs/spotbugs.github.io/pull/12) as example.
+
+## Release to Gradle Plugin Portal
+
+TBU
```

Comment:
> Update site URL with actual plugin update site (not zip, not maven site)

Will be https://spotbugs.github.io/eclipse/

> Unique feature id (distinct from the old one)

For now we haven't change this yet. IMO it's better to change.

> For the maintainers: account at eclipse.org

I have, its mail address is `skypencil@gmail.com`

> Once the plugin entry at marketplace is created, the account owner can invite others to maintain it. They only need an account name at eclipse.org. So all members in the team with eclipse.org account can maintain the plugin entry.

I'll try to handle this later.